src/metabase/api/pulse.clj

```diff
@@ -316,7 +316,8 @@
      :row_count       (:row_count result)
      :col_count       (count (:cols (:data result)))}))
 
-(def ^:private preview-card-width 400)
+;; TODO revert this to 400 before merging to master
```

Comment:
I'll make sure to do this before merging, but perhaps the reviewer should not approve until I've reverted this change, just in case I forgot

Reply:
For context this higher resolution is needed for bug bashing

Reply:
I'm not sure we need to revert that, to be honest. This preview endpoint isn't used for any feature in app I don't think, so it might be ok to just leave the higher resolution.

Reply:
I thought it was used for the image previews for slack messages ([example](https://metaboat.slack.com/archives/C03H6956HNF/p1717426968983259)), or does that come from somewhere else?

Reply:
I think it is safe to drop this now since we won't bug bash the chart